stories/features/feeling-safe.fmf

```diff
@@ -0,0 +1,38 @@
+title: Unsafe scripts executed on tmt runner
+
+story: |
+    As testers or tmt developers, we would like to run user-defined
+    scripts on the tmt runner.
+
+description: |
+    There have been several cases where custom scripts could extend
+    tmt functionality or provide unknowable information. Unfortunately,
```

Comment:
```suggestion
    tmt functionality or provide an unknown information. Unfortunately,
```

Sounds like `unknowable` is usually used for something which is absolutely not possible to know, in any means? Just a nitpick...

Reply:
Fixed in 1ef26810